analytics/filesystem/file_module.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 
 	"github.com/chasex/glog"
+	gglog "github.com/golang/glog"
```

Comment:
Nitpick: can we reference this library as `glog` as it's done in other parts of the code?

Reply:
we have two glogs here 

the code uses two glogs and the existing implementation uses `glog` to refer `github.com/chasex/glog`
```
github.com/chasex/glog
github.com/golang/glog
```

if we use `glog` to refer to `github.com/golang/glog` , then we have to name "github.com/chasex/glog" to `gglog` or other proper name.


let me know your thought on it

Reply:
i agree, let me change it so you can take a another look

Reply:
@guscarreon take another look, i aliased `github.com/chasex/glog` as `clog` so i do not change the critical implementation, the module now uses `glog` to output log information